ide/che-core-ide-api/src/main/java/org/eclipse/che/ide/api/workspace/WorkspaceServiceClient.java

```diff
@@ -116,10 +116,14 @@
      *         workspace ID
      * @param envName
      *         the name of the workspace environment that should be used for start
+     * @param isRecover
```

Comment:
This is a client method and it will always send request with defined `restore` parameter, so server will never use auto-restore behavior if this method called